workflow/resources/datavzrd/spia-template.yaml

```
@@ -119,3 +119,12 @@ views:
               }  
             spec-path: ?input.vega_waterfall
             vega-controls: true
+  pathways_scatter:
+    dataset: spia_table
+    desc: ?f"The y-axis represents the False Discovery Rate (FDR), while the x-axis shows the total pertubation accumulation effect of the pathway for the {wildcards.model} model. 
```

I think mentioning the actual tool used here can be useful context:
```suggestion
    desc: ?f"The y-axis represents the False Discovery Rate (FDR), while the x-axis shows the total pertubation accumulation effect of the pathway that SPIA determined for the {wildcards.model} model. 
```